Mesh_3/include/CGAL/Sizing_field_with_aabb_tree.h

```diff
@@ -17,30 +17,59 @@
 
 #include <CGAL/Profile_counter.h>
 #include <CGAL/Delaunay_triangulation_3.h>
-#include "Facet_patch_id_map.h"
-#include "Get_curve_index.h"
 #include <CGAL/Mesh_3/Protect_edges_sizing_field.h> // for weight_modifier
 
+#include <CGAL/Mesh_3/experimental/Facet_patch_id_map.h>
+#include <CGAL/Mesh_3/experimental/Get_curve_index.h>
+#include <CGAL/Mesh_3/experimental/AABB_filtered_projection_traits.h>
+
 #include <boost/container/flat_set.hpp>
 #if defined(CGAL_MESH_3_PROTECTION_HIGH_VERBOSITY) || defined(CGAL_NO_ASSERTIONS) == 0
 #  include <sstream>
 #  include <boost/format.hpp>
 #endif  // CGAL_MESH_3_PROTECTION_HIGH_VERBOSITY || (! CGAL_NO_ASSERTIONS)
 
-#include "AABB_filtered_projection_traits.h"
-
-template <typename GeomTraits, typename MeshDomain,
-          typename Input_facets_AABB_tree = typename MeshDomain::AABB_tree,
-          typename Get_curve_index_ = CGAL::Default,
-          typename Facet_patch_id_map_ = CGAL::Default
+namespace CGAL {
+/*!
+* @ingroup PkgMesh3DomainFields
+*
+* The class `Sizing_field_with_aabb_tree` is a model of concept `MeshDomainField_3`.
+* It provides a sizing field to be used in the meshing process of a polyhedral surface with features.
+*
+* At each query point, the field value is the minimum of the distances to the
+* surface patches it does not belong to, and the distances to the curves it does belong to.
```

Comment:
> and the distances to the curves it does belong to

What does it mean? As it is, the distance between a query point and a curve it belongs is 0, by definition of "belong to".